test/quizzes_test.py

```diff
@@ -1,20 +1,63 @@
 import unittest
-
 from app.controllers.quizzes_controller import QuizzesController
+from datetime import datetime
 
 class QuizzesTest(unittest.TestCase):
 
     def setUp(self):
         # Run tests on non-production data
-        self.ctrl = QuizzesController('quizzes_test.py')
-        
+        self.ctrl = QuizzesController('quizzes_test.json')
+
     def test_expose_failure_01(self):
         """
-        Implement this function and two more that
-        execute the code and make it fail.
```

Comment:
Test case 01 work fine as expected.

Reply:
I have made a small change to the test case, causing it to fail at quizzes module.